test/persistence-hooks.suite.js

```diff
@@ -1816,6 +1916,147 @@ module.exports = function(dataSource, should) {
       });
     });
 
+    describe('PersistedModel.replaceOrCreate', function() {
```

Comment:
Ummm It seems most of the test cases for hooks are valid for replaceOrCreate as well...